src/Seo.php

```diff
@@ -76,4 +76,53 @@ public static function wpseo_disable_adjacent_rel_links(): bool {
     return get_option(Plugin::L10N . '_wpseo_disable_adjacent_rel_links');
   }
 
+  /**
+   * Adds product GTIN to schema.org structured data.
+   *
+   * @implements woocommerce_structured_data_product
+   */
+  public static function get_product_gtin($data) {
+    global $product;
+
+    if (!$gtin = get_post_meta($product->get_id(), '_' . Plugin::PREFIX . '_gtin', TRUE)) {
+      return $data;
+    }
+
+    switch (strlen(trim($gtin))) {
+      case 8:
+        $gtin_format_type = 'gtin8';
+        break;
+
+      case 13:
+        $gtin_format_type = 'gtin13';
+        break;
+
+      case 14:
+        $gtin_format_type = 'gtin14';
+        break;
+
+      default:
+    }
+
+    $gtin_format_type = 'gtin12';
```

Comment:
I think this line should meant to be the `default` case of the preceding ` switch-case`block, right?

Reply:
Sorry, it was unintentionally moved during the copy and paste. Good catch.